Formula/d/dra.rb

```diff
@@ -1,5 +1,5 @@
 class Dra < Formula
-  desc "Tool to download release assets from GitHub"
+  desc "A command line tool to download release assets from GitHub"
```

Comment:
the best we can do is `CLI tool`, we dont like `A` for brew formula desc

Reply:
I've updated it to `Command-line tool...`. Is it ok?